cla-backend-go/config/config.go

```diff
@@ -67,6 +67,8 @@ type Config struct {
 	CorporateConsoleV1URL string `json:"corporateConsoleV1URL"`
 	CorporateConsoleV2URL string `json:"corporateConsoleV2URL"`
 
+	CLAContributorv2Base string `json:"cla-contributor-v2-base"`
```

Comment:
I have added a terraform entry to set this value. Previously it was not managed by CI/CD. See: https://github.com/LF-Engineering/lfx-easycla-terraform/pull/35. We will want to always consider managing SSM values using terraform moving forward. So, if we encounter SSM values that are not managed, we should take the extra step to add it.

Reply:
Thanks. Had only checked this via the console